locale/ar_IQ/locale.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/ar_IQ/locale.xml
+  *
+  * Copyright (c) 2014-2018 Simon Fraser University
```

Comment:
@vormia, can you change this line and the next to 2019?

Reply:
Done